pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mitxpro-openedx-extensions"
-version = "0.1.1"
+version = "0.2"
```

Comment:
Could you drop this change, we usually have a release-only commit to bump the version separate from the code changes.

Reply:
Done